packages/math-input/src/components/__tests__/integration.test.tsx

```diff
@@ -1,12 +1,11 @@
-import "@testing-library/jest-dom";
 import {
     screen,
     render,
     fireEvent,
     within,
     waitFor,
 } from "@testing-library/react";
-import userEvent from "@testing-library/user-event";
+import {userEvent as userEventLib} from "@testing-library/user-event";
```

Comment:
By aliasing the library import, we can avoid renaming _all_ usages of the `userEvent` throughout these tests. See the `beforeEach()` below.